tests/test_forms.py

```diff
@@ -5,10 +5,10 @@
 
 from debug_toolbar.forms import SignedDataForm
 
-SIGNATURE = "-WiogJKyy4E8Om00CrFSy0T6XHObwBa6Zb46u-vmeYE"
+SIGNATURE = "O7cv5oYzP3ZgjSBrJwdidTUQ5kBWI7oD1xn8fb2vK-w"
 
 DATA = {"value": "foo", "date": datetime(2020, 1, 1, tzinfo=timezone.utc)}
-SIGNED_DATA = f'{{"date": "2020-01-01 00:00:00+00:00", "value": "foo"}}:{SIGNATURE}'
+SIGNED_DATA = f'{{"value": "foo", "date": "2020-01-01 00:00:00+00:00"}}:{SIGNATURE}'
```

Comment:
Detail: I'd prefer it if you reversed the order of `value` and `date` in the dictionary below; this would neatly show that the signature stays the same.

Reply:
Done.